apps/prairielearn/src/server.js

```diff
@@ -86,6 +87,7 @@ if ('h' in argv || 'help' in argv) {
     --config <filename>
     <filename> and no other args        Load an alternative config filename
     --migrate-and-exit                  Run the DB initialization parts and exit
+    --refresh-workspace-hosts-and-exit  Refresh the workspace hosts and exit
```

Comment:
We can bikeshed on this name, but I felt this was preferable to `--mark-all-workspace-hosts-unhealthy-and-exit`?